src/components/TracksTable.vue

```diff
@@ -50,9 +50,16 @@
         <span v-if="props.value">
           {{ props.value | length }}
         </span>
-        <span v-else class="red--text">
-          --:--
-        </span>
+        <VTooltip v-else bottom>
+          <template v-slot:activator="{ on }">
+            <span v-on="on" class="red--text">
+              <VIcon small color="red">mdi-alert</VIcon>--:--
```

Comment:
Maybe a bit more padding between the icon and the `--:--`? They are quite close together now.